app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,13 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @posts = Post.where(author_id: params[:user_id])
+    @comments = Comment.all
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @post = Post.find(params[:id])
+    @users = User.all
+    @comments = Comment.all
```

Comment:
[OPTIONAL] The same thing with the show method. There is no need for the instance variables @comments and @users.